pyproject.toml

```diff
@@ -240,7 +244,7 @@ requires-dist = ["torch", "packaging", "ninja", "causal-conv1d"]
 [[tool.uv.dependency-metadata]]
 name = "deep_ep"
 # This version has to match the version in the commit/rev/tag used
-version = "v1.1.0+e3908bf"
+version = "v1.2.1+bfded34"
```

Comment:
Hey @terrykong @yaoyu-33 can you comment on whether this deepep version bump is ok?

Reply:
Can we try with the old version with the TORCH_CUDA_ARCH_LIST change?